frontend/tsconfig.json

```
@@ -15,7 +15,6 @@
       "@pinia/nuxt",
       "@sentry/core"
     ],
-    "typeRoots": ["./typings", "./node_modules/@types"],
```

Removing the reference to typings would (theoretically) exclude our custom type definitions in that directory from being used.

I actually noticed a similar thing locally, but couldn't reproduce it when running the type check, and was able to fix it by adding `../node_modules/@types` (the monorepo root) as well:

```suggestion
    "typeRoots": ["./typings", "./node_modules/@types", "../node_modules/@types"],
```

However, if our custom extensions of library types are working, then there might not be any need for typings at all :shrug:

---

So I checked locally and the typings are definitely still working!

@obulat It might also be necessary/wise to update `vue-tsc`, I think :thinking:

---

> However, if our custom extensions of library types are working, then there might not be any need for typings at all

I think adding the `typeRoots` helps locating the typings instead of making TypeScript look through all of the possible sources in the whole monorepo. However, I couldn't find the correct path to make it work, and when I take it out, it works 🤷

---

I see! In that case, this appears to work!

```
    "typeRoots": ["./typings", "../node_modules"],
```